pages/api/node-operators/keys-info.ts

```diff
@@ -1,17 +1,120 @@
 import { createNextConnect } from 'modules/shared/utils/createNextConnect'
 import { parseChainId } from 'modules/blockChain/chains'
 import { fetch } from '@lido-sdk/fetch'
+import { CHAINS } from '@lido-sdk/constants'
+import { KeysInfo } from 'modules/motions/types'
+
+export type Module = {
+  id: number
+}
+
+export type KeysInfoOperatorNew = {
+  id: number
+  active: boolean
+  name: string
+  rewardAddress: string
+  totalSigningKeys: number
+  usedSigningKeys: number
+  stakingLimit: number
+  stoppedValidators: number
+  unusedBellowStakingLimit: number
+  unusedOverStakingLimit: number
+  duplicates: string[]
+  invalid: string[]
+}
+
+export type KeysInfoNew = {
+  operators: undefined | KeysInfoOperatorNew[]
+  summary: {
+    moduleId: number
+  }
+}
+
+const requestMainNet = async (chainId: number) => {
+  const data = await fetch(
+    `https://operators.li.fi/api/operators?chainId=${chainId}`,
+  )
+  return data.json()
+}
+const requestTestNet = async (chainId: number) => {
```

Comment:
Maybe it would be better to specify which testnet this request takes data from? For example, by renaming function to `requestGoerliData` or something.